apps/stirling-pdf/0.36.2/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.36.2
+    image: frooodle/s-pdf:0.36.3
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The code appears to be making an update to the `stirling-pdf` service, switching from using version 0.36.2 of an image to version 0.36.3. This is done by modifying the `image` field in the Docker Compose configuration.

To summarize:

1. **Version Update**: Changed from `"frooodle/s-pdf:0.36.2"` to `"frooodle/s-pdf:0.36.3"`.
   - Potential optimization or improvement suggestion: Since both versions contain the same tag (`0.36.3`), there might not be significant performance benefits between them if upgrading doesn't change any underlying logic or dependencies. However, updating can often include bug fixes and security patches that were introduced with newer releases, so it's recommended but non-critical depending on your use case.

No other apparent errors or irregularities are present in this snippet provided for comparison purposes.